app/javascript/packs/components/user-card.js

```diff
@@ -0,0 +1,27 @@
+import React from 'react'
+import PropTypes from 'prop-types'
+import { formatSimpleDate } from '../services/format-date'
+import { translateGender } from '../services/translate-gender'
+
+const UserCard = ({ user }) => {
+  if (!user) return
```

Comment:
Previnir que o usuário exista pode ser feito com `defaultProps` caso n tenha problema carregar esse componente com uma prop sendo um objeto vazio por exemplo

E se for algo que vem do redux já é garantido lá no `defaultState`.

Ou se for o caso de não querer exibir o conteúdo é melhor já n exibir o componente em si, validando isso no componente que chama ele